database/migrations/20250203165654_server.sql

```diff
@@ -0,0 +1,11 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE IF NOT EXISTS counters (
+    key TEXT PRIMARY KEY,
+    value INTEGER NOT NULL DEFAULT 0
+);
+-- +goose StatementEnd
+
+-- +goose Down
+-- +goose StatementBegin
+-- +goose StatementEnd
```

Comment:
I don't actually know anything about gorm, or what these snippets of migration stuff do. I'm assuming you know what this does and that you've made sure it's okay!